src/app/search/map-friendly-values.pipe.ts

```diff
@@ -54,11 +54,11 @@ export class MapFriendlyValuesPipe implements PipeTransform {
     [
       'ToolFile.FileTypeEnum',
       new Map([
-        [ToolFile.FileTypeEnum.PRIMARYDESCRIPTOR, 'Primary Descriptor'],
+        [ToolFile.FileTypeEnum.PRIMARYDESCRIPTOR, 'Config'],
```

Comment:
Does this affect services only? 
It isn't clear to me in the context and it would be weird describing the "first" CWL file a config even if it is correct for Nextflow

Reply:
Have to double check, but I think technically it could be used by all. I'll revert to Primary Descriptor and this can be dealt with later.